README.md

```diff
@@ -1 +1,30 @@
-# cfmr-cf-cli-plugin-network-policy
\ No newline at end of file
+# CF CLI Plugin Network Policy
+
+| Build | State |
+| ---     | ---   |
+| CI | [![CI status](https://github.com/HCL-Cloud-Native-Labs/cfmr-cf-cli-plugin-network-policy/workflows/Main%20CI/badge.svg)](https://github.com/HCL-Cloud-Native-Labs/cfmr-cf-cli-plugin-network-policy/actions?query=workflow%3AMain+CI) |
+
+## About
+This plugin enables App to App direct communication in CFMR
+
+## Usage
+Build image from repo:
+
+```shell script
+# Clone repo and cd into it then...
+docker build --no-cache --tag hclcnlabs/cfmr-cf-cli-plugin-network-policy:1.0.0 .
+```
+
+```shell script
+# Push the image to dockerhub(hclcnlabs)...
+docker push hclcnlabs/cfmr-cf-cli-plugin-network-policy:1.0.0
+```
```

Comment:
Is this still true?

Reply:
This is not relevant now so have removed it.
I will come back to Readme later when the plugin functionality will be completed.